NEWS.md

```diff
@@ -1,15 +1,176 @@
 next:
 
-- Update Gradle to 8.9 thanks to Christophe!
-- replace length = 0 checks with isEmpty() — thanks to Juiceman!
-- do not recommend disabling js helpers in fproxy (these are reviewed)
-- add more resilient plugin list exception handling
-- Remove gc meddling that hasn’t been necessary for many Java releases
-- Add webp filter, thanks to Torusrxxx!
-- Update Japanese localization, thanks to qupo1!
-- Fix build info with gradle daemon, thanks to Bombe!
+- 
+
+1499:
+
+# User changelog
+
+This release mainly brings improvements in five areas:
+
+- improved privacy protection
+- optimized and fixed networking layer
+- support for website authors
+- better user experience
+- clean ups and code purges
+
+## Privacy
+
+The privacy improvements are reduced pings and DNS lookups. These make
+it less likely that a node gets detected by pings it sends for other
+nodes that use DNS addresses to have permanent addresses. Also thanks
+to Torusrxxx SSL support is updated and when setting up a node for
+access via SSL, unencrypted HTTP links now get upgraded to secured
+HTTPS automatically.
+
+## Networking
+
+The networking layer got a fix for very fast nodes. On these the bulk
+queue could be starved when realtime requests were received faster
+than they could be completed. Also bertm fixed bulk backoff getting
+ignored.
+
+Juiceman removed unnecessary boxing and unboxing to reduce CPU load
+and Torusrxxx improved IPv6 handling.
+
+## Website
+
+Torusrxxx also added webp support to the content sanitation (filters),
+so you can now use webp images on freesites and depending on the study
+reduce the storage by around 30%. Minimizing storage is more critical
+on Hyphanet compared to the clearnet, so webp can improve performance
+for all sites.
+
+In addition to webp, Torusrxxx also added support for more CSS
+keywords: border-{top,bottom}-{left,right}-radius, color-scheme,
+dominant-baseline, margin-block, margin-block-end, margin-block-start,
+margin-bottom, margin-inline, margin-inline-end, margin-inline-start,
+margin-right, math-style, padding-block, padding-block-end,
+padding-block-start, padding-inline, padding-inline-end,
+padding-inline-start, padding-right, padding-right, padding-top,
+padding-top, scroll-behavior, scroll-snap-align, scroll-snap-stop,
+scroll-snap-type, text-combine-upright, text-decoration-thickness,
+text-orientation, text-underline-offset.
```

Comment:
Also new HTML elements "bdi","figcaption","figure","main","mark","rp","rt","ruby".
Also a lot of more CSS properties, and fixed a lot of incorrect CSS properties, do I need to compile a full list here?